src/passivedns.c

```diff
@@ -1432,9 +1437,9 @@ int main(int argc, char *argv[])
         /* Look up an available device if non specified */
         if (config.dev == 0x0)
             config.dev = pcap_lookupdev(config.errbuf);
-        olog("[*] Device: %s\n", config.dev);
+        olog("[*] Device: %s (promisc: %d)\n", config.dev, config.promisc);
```

Comment:
I think this looks messy. I would prefer if it didn't print this, especially since this is only implemented when using libpcap.